src/App.js

```diff
@@ -6,7 +6,7 @@ function App() {
     <div className="App">
       
       <header>
-        <h1>Fitness App</h1>
```

Comment:
Why are we adding 3 to Fitness app on the h1?

Reply:
For testing

Reply:
sounds good